react-components/src/components/CacheProvider/AssetMappingCache.ts

```diff
@@ -81,18 +86,29 @@ export class AssetMappingCache {
     revisionId: RevisionId,
     assetIds: CogniteInternalId[]
   ): Promise<Map<AssetId, Node3D[]>> {
-    const assetMappings = await this.getAssetMappingsForAssetIds(modelId, revisionId, assetIds);
     const relevantAssetIds = new Set(assetIds);
 
+    const amountOfChunks = 1;
+    const assetIdsList = Array.from(relevantAssetIds);
+    const chunkSize = Math.round(assetIdsList.length / amountOfChunks);
+    const listChunks = chunk(assetIdsList, chunkSize);
+
+    const allAssetMappingsReturned = listChunks.map(async (itemChunk) => {
+      const assetMappings = await this.getAssetMappingsForAssetIds(modelId, revisionId, itemChunk);
+      return assetMappings;
+    });
+
+    const allAssetMappings = await Promise.all(allAssetMappingsReturned);
```

Comment:
if you see the `_amountOfAssetIdsChunks` which is 1 then it will run only one chunk in total.
If we set the _amountOfAssetIdsChunks to more than 1, then it will split the total in smaller chunks and run in parallel.

Reply:
But from what I can see, there is nothing stopping the code from calling `this.getAssetMappingsForAssetIds` in parallel - when using `await` inside the `async` function in `map`, I think it will go ahead and pick the next chunk in the list and call `getAssetMappingsForAssetIds` without waiting for the first one to succeed.

Reply:
Yes. But only if we change the _amountOfAssetIdsChunks  value. For now with the value being 1, it will request just once.
My thought was to provide this flexibility to change in the future when we have a more robust backend to support more parallel requests. Do you think it is better to have a fixed unique request without this logic that splits in chunks and only change to this current approach when we have sure that the backend is improved?